events/src/main/java/com/app/events/controller/BaseController.java

```diff
@@ -36,7 +37,8 @@
 		return new ResponseEntity<>(errorMessage, HttpStatus.BAD_REQUEST);
 	}
 
-	@ExceptionHandler({ ResourceExistsException.class, DateException.class,
+	@ExceptionHandler({ ResourceExistsException.class,
+		 	ResourceNotFoundException.class, DateException.class,
```

Comment:
resource not found exception treba da vrati status not found sto vraca ako ne stavis nista, jer privi exception u base controlleru hvata njega, jer nije nigde naveden

Reply:
nije tako radilo, pa sam zato dodao.